src/include/function/cast/functions/numeric_limits.h

```diff
@@ -14,6 +14,10 @@ struct NumericLimits {
     static constexpr T minimum() { return std::numeric_limits<T>::lowest(); }
     static constexpr T maximum() { return std::numeric_limits<T>::max(); }
     static constexpr bool isSigned() { return std::is_signed<T>::value; }
+    template<typename V>
```

Comment:
I don't think this comparison is going to work like this.
Can you add a test case for this?

Reply:
This is tested, and it works. It's used in json_utils.cpp:268 to resolve to a minimum integer width.